boards/xtensa/intel_s1000_crb/Kconfig.defconfig

```diff
@@ -63,6 +63,12 @@ config HEAP_MEM_POOL_SIZE
 
 endif # DMA_CAVS
 
+config SPI_0_IRQ_PRI
+	default 3
```

Comment:
you don't need that here, put it in the relevant fixup file (as you for other settings below), and s1000 has no irq priority support afaik (can't recall) so it would be 0.